lms/djangoapps/django_comment_client/tests/test_management_commands_rename_user.py

```diff
@@ -0,0 +1,129 @@
+            "closed": False,
+            "comment_count": 9,
+            "commentable_id": "video_1",
+            "course_id": "MITx/6.002x/2012_Fall",
+            "created_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "historical_abuse_flaggers": [],
+            "last_activity_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "tags_array": [
+                "c#",
+                "2012",
+                "java-sucks",
+            ],
+            "thread_type": "discussion",
+            "title": "Dolore et molestias quia maxime eaque quos voluptatem et.",
+            "updated_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "visible": True,
+            "votes": {
+                "count": 7,
+                "down": [
+                    "1",
+                    "3",
+                    "4",
+                    "5",
+                    "7",
+                ],
+                "down_count": 5,
+                "point": -3,
+                "up": [
+                    "2",
+                    "6",
+                ],
+                "up_count": 2,
+            }
+
+        }
+        self.mongo_comment_id = self.db.contents.insert(comment)
+
+    def seed_mongo_user(self):
+        """
+        Seeds a test  user
+        """
+        user = {
+            "default_sort_key": "date",
+            "notification_ids": [],
+            "external_id": self.sql_user.username,
+            "username": self.sql_user.username,
+        }
+        self.mongo_user_id = self.db.users.insert(user)
+
+    def test_incorrect_num_args(self):
+        """
+        ensures that the command line tools does not work with an incorrect number of arguments
+        """
+        for i in [0, 1, 3]:
+            args = ['arg'] * i
+            self.assertRaises(CommandError, self.rename_cmd.handle, *args)
+
+    @patch('__builtin__.print')
+    def test_successful_change(self, mock_print):
+        """
+        runs the command line tools and ensures successful rename
+        """
+        username_old = self.sql_user.username
+        username_new = username_old + '2'
+        self.assertEqual(0, len(User.objects.filter(
+            username=username_new
+        )))
+        self.rename_cmd.handle(username_old, username_new)
+        self.assertIn('Changed username of user:', mock_print.call_args[0][0])
```

Comment:
We can actually remove this line; if the call to `self.rename_cmd.handle` does not throw an exception, then we know implicitly that it succeeded. This way, we the tests are less brittle as they don't depend on the text output.